includes/class-omise-money.php

```diff
@@ -0,0 +1,56 @@
+<?php
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * @since 3.5
+ */
+class Omise_Money {
+	/**
+	 * @var array
+	 */
+	private static $subunit_multiplier = array(
+		'AUD' => 100,
+		'CAD' => 100,
+		'CHF' => 100,
+		'CNY' => 100,
+		'DKK' => 100,
+		'EUR' => 100,
+		'GBP' => 100,
+		'HKD' => 100,
+		'JPY' => 1,
+		'MYR' => 100,
+		'SGD' => 100,
+		'THB' => 100,
+		'USD' => 100
+	);
+
+	/**
+	 * @param  int|float $amount
+	 * @param  string    $currency
+	 *
+	 * @return int
```

Comment:
I think we should return fractional subunits if their input results in this. We shouldn't quietly be removing money. Whilst this is almost insignificant on a single conversion, repeated conversions (I can't think of a good example scenario - maybe micropayments?) over time could result in considerable discrepancies

Reply:
If we remove stuff quietly like this, the developer may not realise their initial mistake as it is being swept under the carpet. Better to let them get a nasty error when they try to charge with the fractional subunits